packages/tests/server/interop/middleware.test.ts

```diff
@@ -497,11 +497,10 @@ test('omitting ctx in next() does not affect a previous middleware', async () =>
       })
       .query('test', {
         resolve({ ctx }) {
-          expectTypeOf(ctx).toEqualTypeOf<
-            OriginalContext & {
-              user: User;
-            }
-          >();
+          expectTypeOf(ctx).toEqualTypeOf<{
```

Comment:
![CleanShot 2023-12-18 at 19 22 42@2x](https://github.com/trpc/trpc/assets/51714798/f8d55632-cd77-4eb8-9d85-56aef5469b88)

this is from main, it's the same. I guess the matcher (expect-type) has changed and gotten stricter?